qa/multi-cluster-search/src/test/resources/rest-api-spec/test/multi_cluster/10_basic.yml

```diff
@@ -115,6 +115,39 @@
   - match: { aggregations.cluster.buckets.0.key: "local_cluster" }
   - match: { aggregations.cluster.buckets.0.doc_count: 5 }
 
+  # once more, this time with a pipeline agg
+  - do:
+      search:
+        rest_total_hits_as_int: true
+        index: test_index,my_remote_cluster:test_index
+        body:
+          seq_no_primary_term: true
+          aggs:
+            cluster:
+              terms:
+                field: f1.keyword
+              aggs:
+                s:
```

Comment:
Should we add a non-top-level pipeline agg just to confirm they aren't affected?

Reply:
I figured I'd get it in my next PR *about* non-top-level pipeline aggs, but I'm happy to do it now!